app/inc/constants.php

```diff
@@ -28,7 +28,8 @@
 }
 
 // Special modes for the app
-define('DEBUG', (strstr(VERSION, 'dev') || isset($_GET['debug'])) ? true : false);
+define('DEBUG',     strstr(VERSION, 'dev') || isset($_GET['debug']));
+define('LOCAL_DEV', isset($server_config['dev']) && $server_config['dev']);
```

Comment:
actually, dev=false is already in config.ini-dist:
https://github.com/mozfr/transvision/blob/master/app/config/config.ini-dist#L32